dietpi/dietpi-software

```diff
@@ -5293,6 +5304,34 @@ Package: wireguard wireguard-dkms wireguard-tools\nPin: release n=bullseye\nPin-
 
 		fi
 
+		software_id=177 # Firefox Sync Server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# Dependencies
+			DEPS_LIST='python-virtualenv libffi-dev libssl-dev'
+			# - Pre-compiling required on ARM
+			(( $G_HW_ARCH < 10 )) && DEPS_LIST+=' libffi-dev libssl-dev'
+
+			INSTALL_URL_ADDRESS='https://github.com/mozilla-services/syncserver'
+
+			# If there is an existing installation update it as per Mozilla's instructions, else fresh install
+			if [[ -d /etc/firefox-sync ]]; then
+				cd /etc/firefox-sync
+				git stash
+				git pull
+				git stash pop
+			else
+				git clone "$INSTALL_URL_ADDRESS" $G_FP_DIETPI_USERDATA/firefox-sync
```

Comment:
$G_FP_DIETPI_USERDATA should not be used as temporary download dir. DietPi scripts navigate into `/tmp/$G_PROGRAM_NAME`, here `/tmp/DietPi-Software`, hence a tmpfs which is perfectly fine as temporary download dir. So it could be:
```
git clone "$INSTALL_URL_ADDRESS"
mv syncserver /etc/firefox-sync
```
... respectively an alternative download method (not Git) and alternative target dir as suggested above.